tools/cldr-code/src/main/java/org/unicode/cldr/util/XPathParts.java

```diff
@@ -1273,4 +1262,14 @@ private XPathParts removeAttribute(String attribute) {
         }
         return this;
     }
+
+    public List<String> getPathAttributes() {
```

Comment:
Minor, and we could do latter.

We should call this getPathAttributeValues, because it is the attribute **values** that we are getting, not the attributes.

Reply:
Or just getAttributeValues now that it's an instance method -- there's already a getAttributeValue (singular) method